packages/slice-machine/tests/lib/mock/Slice.test.js

```diff
@@ -1,5 +1,4 @@
 import MockSlice from "../../../lib/mock/Slice";
```

Comment:
Why this file is even back here? The library generate the mocks, it makes no sense to test again the output of the lib... seems like overtesting here.

Reply:
> Why this file Slice.test.js is even back here
Because the function still exists, why did you delete it in the first place? 
Chances are it was added there because something broke before.

>The library generate the mocks, it makes no sense to test again the output of the lib
Didn't you say the lib could break, or is likely to change?

> seems like over-testing here
I honestly I think you would any testing is too much for you.

> it’s like testing that React actually renders components
oddly enough react comes with a just utils package for that https://reactjs.org/docs/test-utils.html.


If you really worried about over testing there are practices for that, but those practice involve writing the engineer writing the code to write the tests for the features they are working on. That did not happen on this project so we have to do what we can, cleaning as we go.

Reply:
there's logic done from slice-machine itself. The purpose of the test is not to check the library is generating the right data, it's to test that we are creating the right objects and passing the right arguments to the library functions. There's a bunch of object manipulation, object deconstruction (`...`), array methods (`.reduce`, `.map`) and we should test them :) 
<img width="697" alt="image" src="https://user-images.githubusercontent.com/69144671/181066452-a4a04150-5450-48df-9124-811ec607c773.png">

Reply:
it’s true but since everything is typed, I consider that the output is correctly built, that’s how I would consider it here. But you’re right there is still a bit of logic here. And the Slice.test.ts that we had before was about testing what was generated and I consider that part useless, it should be part of the lib. 
But testing the logic that builds the config for the lib makes sense. Thanks for that explanation @sintinilorenzo